internal/pkg/skuba/kubernetes/versions.go

```diff
@@ -81,17 +85,44 @@ type ClusterAddonsKnownVersions = func(clusterVersion *version.Version) AddonsVe
 
 var (
 	supportedVersions = KubernetesVersions{
+		"1.18.0": KubernetesVersion{
+			ComponentHostVersion: ComponentHostVersion{
+				KubeletVersion:          "1.18.0",
+				ContainerRuntimeVersion: "1.17.0",
+			},
+			ComponentContainerVersion: ComponentContainerVersion{
+				APIServer:         &ContainerImageTag{Name: "api-server", Tag: "v1.18.0"},
+				ControllerManager: &ContainerImageTag{Name: "controller-manager", Tag: "v1.18.0"},
+				Scheduler:         &ContainerImageTag{Name: "scheduler", Tag: "v1.18.0"},
+				Proxy:             &ContainerImageTag{Name: "proxy", Tag: "v1.18.0"},
+				Etcd:              &ContainerImageTag{Name: "etcd", Tag: "3.4.3"},
+				CoreDNS:           &ContainerImageTag{Name: "coredns", Tag: "1.6.7"},
+				Pause:             &ContainerImageTag{Name: "pause", Tag: "3.2"},
+				Tooling:           &ContainerImageTag{Name: "skuba-tooling", Tag: "0.1.0"},
+			},
+			AddonsVersion: AddonsVersion{
+				Cilium:        &AddonVersion{"1.5.3", 2},
+				Kured:         &AddonVersion{"1.3.0", 4},
+				Dex:           &AddonVersion{"2.19.0", 5},
```

Comment:
Currently we are going to update Dex to version `2.23.0` there is still present `2.16.0`
https://github.com/SUSE/avant-garde/issues/1466